src/Geno/C++/GUI/Widgets/TextEdit.h

```diff
@@ -16,6 +16,11 @@
  */
 
 #pragma once
+
+#if defined( _WIN32 )
+#undef ReplaceFile
+#endif // _WIN32
+
```

Comment:
This needs to be done after the includes

Reply:
Okay